indy_node/server/domain_req_handler.py

```diff
@@ -628,6 +629,8 @@ def handleGetRevocRegDelta(self, request: Request):
                 if req_ts_from and accum_from.value:
                     reply[STATE_PROOF_FROM] = accum_from.proof
                     reply[VALUE][ACCUM_FROM] = accum_from.value
+                    reply[VALUE][SEQ_NO_FROM] = accum_from.seq_no
```

Comment:
So, did we change the format of Reply? Does master have exactly the same fields and names?

Reply:
Maybe we can just add SEQ_NO to ACCUM txn value to avoid changes in SDK to support the new format...